widget_driver_generator/test/code_providers/test_driver_code_provider_test.dart

```diff
@@ -0,0 +1,210 @@
+import 'package:test/test.dart';
+import 'package:widget_driver_generator/src/code_providers/test_driver_code_provider.dart';
+import 'package:widget_driver_generator/src/models/annotated_element.dart';
+
+void main() {
+  group('TestDriverCodeProvider:', () {
+    test('Builds with no elements', () {
+      final codeProvider = TestDriverCodeProvider(
+        methods: [],
+        properties: [],
+        fields: [],
+        driverClassName: 'ExampleDriver',
+      );
+      const expected = '''
+class _\$TestExampleDriver extends TestDriver implements ExampleDriver {
+  
```

Comment:
just curious: why do we need those multiple empty lines?

Reply:
Because I wanted to add the different fields in a more readable way in the `testDriverClass` getter. So I added new lines and tabs there. Which in Prod get formatted on generation, however they don't in tests.